src/en/developers/google-summer-of-code/index.md

```diff
@@ -26,33 +26,34 @@ get started with contributions](https://ceph.io/en/developers/contribute/).
 
 <hr class="hr">
 
-## Improving Shaman the Builder
+## Shaman Scheduling for Success
 
 **Mentor name(s):** Zack Cerza, Kamoltat (Junior) Sirivadhna
 Aishwarya Mathuria, Vallari Agrawal
 
-**Mentor email(s):** zcerza@ibm.com, ksirivad@ibm.com, amathuri@ibm.com,
-vallariag@ibm.com
+**Mentor email(s):** zack1@ibm.com, ksirivad@ibm.com, aishwarya.mathuria@ibm.com,
+vallari.agrawal@ibm.com
 
 **Difficulty:** Medium
 
 **Project Hours:** 350
 
 **Skills needed:** Shell, Python
-**Subcomponent of Ceph:** [Build System](https://github.com/ceph/ceph-build)
+
+**Subcomponent of Ceph:** [Ceph Build System](https://github.com/ceph/ceph-build), [Ceph Integration Test Framework](https://github.com/ceph/teuthology)
 
 **Description of project:**
-Ceph's build system is far from perfect and have encoutered several
-problems in the past. Your job is to work with Ceph's
-Infrastructure + Teuthology Team to improve the current build system.
+Enable ceph developers to preset Teuthology commands (scheduling) when pushing a feature branch to Shaman.
```

Comment:
This is the previous work done on auto-scheduling: 
Jenkins would be sending notifications for finished builds to teuthology-api: https://github.com/VallariAg/ceph-build/commit/217f080a45c00a07829be9c0ce51057f23b27ddc 
And t-api will handle the queuing commands and scheduling them: https://github.com/ceph/teuthology-api/pull/24